config/routes.rb

```diff
@@ -7,6 +7,7 @@
   delete '/logout' => 'session#destroy'
   get '/register' => 'users#new'
   post '/register' => 'users#create'
+  get '/dashboard' => 'users#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.